src/mempool/mempool_test.rs

```diff
@@ -172,9 +175,39 @@ mod tests {
         assert!(!valid)
     }
 
+    #[tokio::test]
+    async fn test_rate_limits_applied() {
+        let (mut engine, _, mut mempool, _, _, _, _) = setup(None, true);
```

Comment:
Hmm, the only difference is that we pass true here? Can we just pass trie everywhere and remove this test? What do we gain from setting it to false in other tests?

Reply:
It's a bit annoying to have to think about the rate limits in all the tests. They all fail right now if I enable the rate limits.